File: Backend/DA_QLCafe.API/Models/KhachHang.cs
namespace DA_QLCafe.API.Models
{
    public class KhachHang
    {
        public int Ma { get; set; }
        public string HoTen { get; set; } = string.Empty;
        public string? SoDienThoai { get; set; }
        public string? Email { get; set; }
        public int DiemTichLuy { get; set; }
        public DateTime? NgaySinh { get; set; }
    }
}

File: Frontend/src/interfaces/KhachHang.ts
export interface KhachHang {
  ma: number;
  hoTen: string;
  soDienThoai: string | null;
  email: string | null;
  ngaySinh: string | null;
  diemTichLuy: number;
}

export interface KhachHangLoyalty extends Omit<KhachHang, 'ma' | 'hoTen' | 'ngaySinh'>{
    maKhachHang: number;
    tenKhachHang: string;
    hangKhachHang: string;
    ngaySinh?: string | null;
}